js/js.js

```diff
@@ -1,3 +1,28 @@
-$(function () {
-  $('[data-toggle="tooltip"]').tooltip()
-})
\ No newline at end of file
+$(document).ready(function() {
+
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
+  });
+
+  // open modal window if a hash is found in URL
+  if(window.location.hash) {
+    $(window.location.hash).modal('show');
+    // add direct link to modal window
+    $(".link").empty();
+    $(".link").append("<small>Link to this command: "+window.location.href+"</small>");
```

Comment:
I presume something like
`$(".link").append("<small>Link to this command: <a href="+window.location.href+">"+window.location.href+"</a></small>");`
should work. In order to respect the HTML5 standard, also additional " around the path should be added. I have to check how to do this in that language I never use.